src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -926,6 +932,40 @@ protected Date extractDate(Document doc) {
             }
         }
 
+        // computerweekly.com - extraction from javascript code
+        elems = doc.select("script[type=text/javascript]");
+        for (Element e : elems) {
+            if (e.toString().contains("main-article-author-date")) {
+                Matcher matcher = COMPUTER_WEEKLY_DATE_PATTERN.matcher(e.toString());
+                if(matcher.find()) {
+                    dateStr = matcher.group(1);
+                    Date parsedDate = parseDate(dateStr);
+                    if (DEBUG_DATE_EXTRACTION) {
+                        System.out.println("RULE-script[type=text/javascript]");
+                    }
+                    if (parsedDate != null) {
+                        return parsedDate;
+                    }
+                }
+            }
+        }
+
+        // http://www.business-money.com/announcements/catalyst-achieves-300m-funding-milestone
+        // There is no metadata at all to identify the date other than complete css selector
+        // This might be really bad hack but we don't have other option
+        elems = doc.select("body > div:nth-child(7) > div > div.col-sm-9 > div > h1 > span > p > span");
```

Comment:
I wonder if we can do as a last resort some sort of search for common dates format in the article body. Ideally we don't to get this specific when defining rules (although in some cases we may have not option).